massa-execution-worker/src/interface_impl.rs

```diff
@@ -512,6 +613,46 @@ impl Interface for InterfaceImpl {
         Ok(keys)
     }
 
+    fn get_op_keys_wasmv1(&self, prefix: &[u8]) -> Result<Vec<Vec<u8>>> {
+        // compute prefix range
+        let prefix_range = if !prefix.is_empty() {
+            // compute end of prefix range
+            let n_keep = prefix
+                .iter()
+                .enumerate()
+                .rev()
+                .find_map(|(i, v)| if v < &255 { Some(i + 1) } else { None })
+                .unwrap_or(0);
+            let mut prefix_end = prefix[..n_keep].to_vec();
```

Comment:
This seems a bit too complicated for what it does, probably a better way to do it but I don't care so much as long as it works.

Maybe something like:
```rust
let mut prefix_end = prefix.to_vec();
while let Some(255) = end_range.last() {
    end_range.pop();
}
```
I know iterators are cheap but you're chaining 3 of them for a pretty simple operation. Its also easier to read.

Reply:
It is better indeed!